### DISCUSSION TITLE
Does this strip_comments have to be True by default?

### DISCUSSION THREAD

**Discussion by singhrajk** [apache/superset#22701] [General]
In continuation to this [issue](https://github.com/apache/superset/issues/22459), wanted to know if it makes sense to disable the strip_comments, because we want to send comments from superset, as we have some special comments based on which, on the server side we make some decisions.

Also, this is contrary to the [base](https://github.com/apache/superset/blob/26545a26d195b79ab6e838631e86cb0bf20f3ced/superset/db_engine_specs/base.py#L282) enginespec where comments are enabled and I guess trino enginespec is not overriding that meaning (it should also allow comments).

Can comments be enabled on SQLLab?

cc: @rusackas @betodealmeida

**Comment by rusackas** [comment]
I suspect that for the vast majority of use cases, the comments are unnecessary, and perhaps even problematic. Rather than flipping it, I could see a case for having this reflect a setting in `config.py` then it's up to you and your deployment, and we could leave the default as-is.

**Reply by singhrajk** [reply]
Agreed. But wanted to understand if enabling comments will bring in some unwanted bug that might hamper the user experience?

**Reply by rusackas** [reply]
Not sure. Maybe @villebro or @betodealmeida recall the rationale for stripping them out in the first place.

**Reply by singhrajk** [reply]
@villebro @betodealmeida could you shed some light on this? 🙇‍♂️

**Reply by rusackas** [reply]
CC @eschutho for good measure.
